dotCMS/src/main/java/com/dotcms/rest/api/v1/contenttype/ContentTypeForm.java

```diff
@@ -0,0 +1,127 @@
+package com.dotcms.rest.api.v1.contenttype;
+
+import com.dotcms.contenttype.model.type.ContentType;
+import com.dotcms.contenttype.transform.contenttype.JsonContentTypeTransformer;
+import com.dotcms.repackage.com.fasterxml.jackson.core.JsonParser;
+import com.dotcms.repackage.com.fasterxml.jackson.core.JsonProcessingException;
+import com.dotcms.repackage.com.fasterxml.jackson.databind.DeserializationContext;
+import com.dotcms.repackage.com.fasterxml.jackson.databind.JsonDeserializer;
+import com.dotcms.repackage.com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.dotcms.repackage.com.google.common.annotations.VisibleForTesting;
+import com.dotmarketing.business.DotStateException;
+import com.dotmarketing.exception.DotRuntimeException;
+import com.dotmarketing.util.json.JSONArray;
+import com.dotmarketing.util.json.JSONException;
+import com.dotmarketing.util.json.JSONObject;
+import org.jetbrains.annotations.NotNull;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+/**
+ * {@link ContentTypeResource}'s form
+ */
+@JsonDeserialize(using = ContentTypeForm.ContentTypeFormDeserialize.class)
+public class ContentTypeForm  {
+
+    private List<ContentTypeFormEntry> entries;
+
+    public ContentTypeForm(List<ContentTypeFormEntry> entries) {
+        this.entries = entries;
+    }
+
+    public Iterable<ContentTypeFormEntry> getIterable() {
+        return entries;
+    }
+
+    public ContentType getContentType() {
+        return entries.get(0).contentType;
+    }
+
+    public List<String> getWorkflowsIds() {
+        return entries.get(0).workflowsIds;
+    }
+
+    public static final class ContentTypeFormDeserialize extends JsonDeserializer<ContentTypeForm> {
+
+        @Override
+        public ContentTypeForm deserialize(JsonParser jsonParser, DeserializationContext deserializationContext) throws IOException, JsonProcessingException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'com.dotcms.repackage.com.fasterxml.jackson.core.JsonProcessingException' which is a subclass of 'java.io.IOException'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 158 characters long line (which is greater than 150 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00103)